app/services/import/persist-licence.service.js

```diff
@@ -156,6 +167,38 @@ async function _persistCompany (trx, updatedAt, company) {
     ])
 }
 
+async function _persistContact (trx, updatedAt, contact) {
+  return ContactModel.query(trx)
+    .insert({ ...contact, updatedAt })
+    .onConflict('externalId')
+    .merge([
+      'salutation',
```

Comment:
Should this update the data source to nald ? what if this has changed in wrls. Might be a none issue as the test helper sets this as wrls.